api/src/certification/course/application/certification-course-controller.js

```diff
@@ -28,16 +28,14 @@ const unreject = async function (request, h, dependencies = { events }) {
   return h.response().code(200);
 };
 
-const updateJuryComments = async function (request, h, dependencies = { assessmentResultSerializer }) {
+const updateJuryComment = async function (request, h, dependencies = { juryCommentSerializer }) {
   const certificationCourseId = request.params.id;
-  const deserializedAssessmentResultComments = await dependencies.assessmentResultSerializer.deserialize(
-    request.payload,
-  );
+  const assessmentResultCommentByJury = await dependencies.juryCommentSerializer.deserialize(request.payload);
```

Comment:
```suggestion
  const juryComment = await dependencies.juryCommentSerializer.deserialize(request.payload);
```

Je suis en phase avec le fait que le deserializer ne génère pas de AssessmentResult; je propose de propager ce changement au nom de cette variable, ain,si qu'au paramètre du usecase

Reply:
De mémoire on s'est arrêté là car justement le usecase génère un nouvel assessmentResult pour mettre les commentaires...
On ne savait pas spécialement comment refléter ça autrement qu'en laissant le nom dans la variable. On peut décider tous ensemble :) 

Je me dis que la bonne chose à faire, c'est sans doute plus de ne plus générer un assessment result à chaque fois, si comme on l'a pressenti l'historique n'a pas d'importance => Epic de refacto

Reply:
Après échange avec Stephen, on se dit que c'est bien de l'évoquer tous ensemble demain, qu'on soit bien alignés :)

Reply:
Il y a un loup, mais il semble plus large que le scope de cette PR.
On s'est mis d'accord pour laisser en l'état et attendre de voir si on retouche à quelle entité métier porte le commentaire pour avoir un nom de usecase un peu plus transparent.

Reply:
Vu ensemble j'ai dis que c'était pas bloquant un nommage non plus pour faire passer la PR ^^